### DISCUSSION TITLE
Supprimer plusieurs champs d'un modèle en une fois

### DISCUSSION THREAD

**Discussion by MathildeHuguin** [Inist-CNRS/lodex#2061] [Ideas]
**Constat**
Dans le cas de l'import d'un modèle existant dans une instance : il faut entrer individuellement dans chaque champ du modèle (que ce soit sur la page d'accueil, la ressource principale ou la page dédiée aux graphiques) pour les supprimer. 

**Idée d'amélioration**
Ajouter **(i)** une case pour pouvoir sélectionner tous les champs que l'on souhaite supprimer et **(ii)** un/des bouton(s) `supprimer tout` ou `supprimer uniquement la sélection` permettrait de gagner du temps lorsque l'on souhaite re-travailler sur un modèle existant.

**Comment by camilledesalabert** [comment]
Dans le même ordre d'idée, pouvoir afficher ou non en 1 action toute une liste de champs sélectionnés
